docker-compose.yml

```diff
@@ -1,62 +1,76 @@
-version: '3.2'
-services:
-  db:
-    image: postgres:10
-    environment:
-      - POSTGRES_DB=hitech_apd
-      - POSTGRES_PASSWORD=cms
-      - PGDATA=/pgdata
-    volumes:
-      - pgdata:/pgdata
-    expose:
-      - 5432
-    ports:
-      - 54325:5432
-
-  web:
-    build: ./web
-    image: cms-eapd/web
-    environment:
-      - API_URL=http://localhost:8081
-      - OKTA_DOMAIN=${OKTA_DOMAIN}
-      - OKTA_SERVER_ID=${OKTA_SERVER_ID}
-      - OKTA_CLIENT_ID=${OKTA_CLIENT_ID}
-    volumes:
-      - type: bind
-        source: ./web
-        target: /app
-        consistency: delegated
-      - /app/node_modules
-    ports:
-      - 8080:8001
-
-  api:
-    build: ./api
-    image: cms-eapd/api
-    depends_on:
-      - db
-    environment:
-      - PORT=8000
-      - DEV_DB_NAME=${DEV_DB_NAME:-hitech_apd}
-      - NODE_ENV=development
-      - DISABLE_SAME_SITE=yes
-      - OKTA_DOMAIN=${OKTA_DOMAIN}
-      - OKTA_SERVER_ID=${OKTA_SERVER_ID}
-      - OKTA_CLIENT_ID=${OKTA_CLIENT_ID}
-      - OKTA_API_KEY=${OKTA_API_KEY}
-      - JWT_SECRET=${JWT_SECRET}
-      - MONGO_DATABASE=${MONGO_DATABASE}
-      - MONGO_URL=${MONGO_URL}
-      - CYPRESS_TESTS=${CYPRESS_TESTS}
-    volumes:
-      - type: bind
-        source: ./api
-        target: /app
-        consistency: delegated
-      - /app/node_modules
-    ports:
-      - 8081:8000
-      - 9229:9229
-
-volumes:
-  pgdata:
+version: '3.2'
+services:
+  db:
+    image: postgres:10
+    environment:
+      - POSTGRES_DB=hitech_apd
+      - POSTGRES_PASSWORD=cms
+      - PGDATA=/pgdata
+    volumes:
+      - pgdata:/pgdata
+    expose:
+      - 5432
+    ports:
+      - 54325:5432
+
+  mongo:
+    image: mongo:4

```

Comment:
can you switch this to `mongo:latest`

Reply:
I have updated it to latest as requested